fastmath.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+import argparse
+import ast
+import importlib
+import pathlib
+
+
+def get_njit_funcs():
+    """
+    Identify all njit functions
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    njit_funcs : list
+        A list of all njit functions, where each element is a tuple of the form
+        (module_name, func_name)
+    """
+    ignore_py_files = ["__init__", "__pycache__"]
+
+    pkg_dir = pathlib.Path(__file__).parent / "stumpy"
```

Comment:
Tell me what you think. I almost feel like `pkg_dir` should be an input parameter. So, in `__main__`, we pass our desired directory into this function or possibly make it a required command line input parameter. This will help ensure that this function can be generalized in the future (if we need to use it for another package).

Reply:
I agree!

This is similar to the challenge I am trying to handle at work. There is a module that contains not just the process but also some values that are related to the input data. There was no problem initially. But then the scope of project became bigger and the same process was needed to be applied to different data that are somewhat different. So, now, I have to decouple the process from those values!

So... I agree. Although we may not use it again later, it is still better to have just the "process" in the function